.pi/skills/code-review/SKILL.md

```
@@ -35,124 +38,99 @@ coderabbit --version 2>/dev/null || echo "NOT_INSTALLED"
-6. Repeat until clean or only info-level issues remain
+Create a task list for Critical + Warning items.
 
-### 5. Review Specific Changes
+### 5. Fix Loop (Development Integration)
 
-**Review only uncommitted changes:**
+When the user requests implement+review, or autonomously before committing a non-trivial change:
 
-```bash
-cr review --agent -t uncommitted
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)